package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "@actions/cache": "^1.0.6",
     "@actions/core": "^1.2.6",
+    "node-fetch": "^2.6.1",
```

Comment:
Oh, I see, it is _already_ in `package-lock.json`, thanks to 47b9f41542ba6db9d7aeb27cd2133a7d96088280 (which added `@actions/cache`)...

Reply:
Exactly, but having it in the `package.json` as well ensures that you'll have it, even when `@actions/cache` would remove it in the future 👍🏼